NEWS

```diff
@@ -9,6 +9,7 @@ Enhancements
 Bug fixes
  - Fix Outlook crashes when modifying the view of a folder
  - Fix server side crash when reading some recurrence appointments
+ - Be able to have group mail as to, cc, bcc or from
```

Comment:
Changes must be in chronological order, newer to older (so always add first line). This way will be easier when we release new packages (it's order DESC because the file itself is DESC).